gooey-gui/app/styles/custom.css

```
@@ -13,7 +13,7 @@
 .streamlit-like-btn:hover,
 .streamlit-like-btn:active {
   cursor: pointer;
-  font-family: avenir-lt-w01_85-heavy1475544,avenir-lt-w05_85-heavy,"Space Grotesk", sans-serif;
+  font-family: avenir-lt-w05_85-heavy,"Space Grotesk", sans-serif;
```

Stylelint (17.14.1)

[error] Expected quotes around "avenir-lt-w05_85-heavy" (font-family-name-quotes)

(font-family-name-quotes)